journal/modernized/03-chapter-2.adoc

```diff
@@ -1,8 +1,8 @@
 == Section II.
 
-Reciting several difficult exercises he went through, both inwardly and outwardly,
-between his convincement and the setting up of a public meeting at Lurgan.
-//again, another sub-heading of sorts?
+[.chapter-synopsis]
+* Several Difficult Exercises He Went Through, both Inwardly and Outwardly,
+* between His Convincement and the Setting Up of a Public Meeting at Lurgan
```

Comment:
Since this is just one sentence, I don't think we can break it up into two different chapter synopsis pieces and start a new one with the word _between_.  I think in this case it all has to be just one long one. Like 

`[.chapter-synopsis]`
`* Several Difficult Exercises He Went Through, both Inwardly and Outwardly, between His Convincement and the Setting Up of a Public Meeting at Lurgan`

Reply:
I'm not even sure why I did that, it really doesn't make sense!!